pkg/distributor/distributor_test.go

```diff
@@ -413,6 +414,17 @@ func TestDistributor_PushIngestionRateLimiter(t *testing.T) {
 				{metadata: 1, expectedError: httpgrpc.Errorf(http.StatusTooManyRequests, "ingestion rate limit (5) exceeded while adding 0 samples and 1 metadata")},
 			},
 		},
+		"unhappy ingestors: rate limit should be unaffected when ingestion fails": {
```

Comment:
They're called "ingesters" not "ingestors". It's a very typical typo.

```suggestion
		"unhappy ingesters: rate limit should be unaffected when ingestion fails": {
```